src/leveled_sstblock.erl

```diff
@@ -0,0 +1,799 @@
+%% -------- SST Block Functions ---------
+%%
+%% Functions to serialise and deserialise blocks in leveled_sst 
```

Comment:
This comment suggests an API with serialize and deserialize functions.
`serialize_block` says it all, but it is not clear that the other functions deserialize or possibly partly deserialize.
Possibly it would be nice to reflect that in their name, such as `get_all_deserialized`.
In particular when it comes to version 0 blocks, that is exactly what get_all would do.